Display.cpp

```diff
@@ -0,0 +1,87 @@
+#include "Display.h"
+#include <iostream>
+
+using namespace std;
+
+Display::Display(vector<LogLine>& set) : dataset(set) {}
+
+void Display::printPV(string pid, string vid)
+{
+    for(auto index : dataset)
+    {
+        if(index.getProcessId() == pid && index.getThreadId() == vid)
```

Comment:
use parenthesis to diff between two conditions

Reply:
done